web/docs/introduction/getting-started.md

```diff
@@ -50,11 +50,11 @@ Check [More Details](#more-details) section below if anything went wrong, or if
 
 ------
 
-## More details 
+## More details
 
 ### Requirements
 
-You must have Node.js (and NPM) installed on your machine and available in `PATH`. We rely on the latest Node.js LTS version (currently `v18.14.2`).
+You must have a sufficiently up-to-date version of Node.js (and NPM) installed on your machine and available in `PATH`.
```

Comment:
This sounds as complex language, let's keep it simple e.g.
```suggestion
Your Node.js version should be at least 20 and available in `PATH`. This means we are okay with you using newer version than Node 20, but not older version like Node 18.
```

Reply:
I didn't want to mention a specific version of Node, so we don't have to maintain this part of the docs and it doesn't go stale. I wrote something on my own and then had ChatGPT touch it up a bit :D, I thought it is pretty readable / not overly complex!

Reply:
I made it very simple:
```
You must have Node.js (and NPM) installed on your machine and available in `PATH`.
Version of Node.js must be >= 18.
```